cypress/support/step_definition/team/team.spec.js

```diff
@@ -0,0 +1,31 @@
+import Team from '../../pages/team/pageTeam'
+
+const selectTeam = new Team()
```

Comment:
here also object name should be same as class name.

Reply:
Adjusted, can you please recheck.